recipe/meta.yaml

```diff
@@ -18,27 +18,44 @@ requirements:
   host:
     - python
     - pip
+    - setuptools
+    - wheel
```

Comment:
```suggestion
```
poetry-core is a build backend, and it has its own WheelBuilder. In that case, setuptools and wheel are redundant here